### ISSUE TITLE
Weird blank screen after resume bug on 6.4.1

### ISSUE THREAD

**Issue by xuanruiqi** [CachyOS/kernel-patches#16]
So, I just updated to 6.4.1 recently, and there's an annoying bug: whenever I resume from suspend, the screen is blank and I couldn't do anything but SysRq reboot my machine. I'm sure it's not an X or GDM problem as I couldn't get a TTY either. (My graphics card is NVIDIA and I'm using the proprietary drivers, but since I can't get a TTY either I think there's nothing to do with the drivers.)

Not sure what's causing the problem, but the stock Arch linux v6.4.1 kernel works alright for me.

**Comment by xuanruiqi** [comment]
No, I think it is not related to DDCCI. I do not have DDCCI enabled and still get this bug.
I even tried rebuilding my kernel with the stock config and it still has this bug. Meanwhile neither Cachy 6.4.0 nor Arch stock 6.4.1 have this bug, so it must be a regression in Cachy 6.4.1.

**Comment by kotarf** [comment]
For what it's worth, suspend was functional for me in cachy 6.3.9 and regressed in 6.4.x. I haven't tried stock Arch yet. I have experimented with various kernel flags changing sleep states and blacklisting modules, but suspend (and hibernate) remain completely non-functional.

**Comment by xuanruiqi** [comment]
6.4.2 does not fix bug

**Comment by xuanruiqi** [comment]
I can't replicate the issue on Intel graphics machines or Optimus dual graphics machines (when running with Intel as main graphics provider). The issue is still present with latest CachyOS kernel and latest NVIDIA drivers.

**Comment by ptr1337** [comment]
This should be fixed since 6.4.6.

Can you check?

**Comment by ptr1337** [comment]
> Yeah, this was fixed. Thanks! I'm curious what was the fix. Was it a bug in the patches or the upstream source?

This was a result of a scheduler merge from 6.6, but those got also removed now
We were not able to identify the exact commit but gladly it was not the EEVDF patch.
